optimum/exporters/onnx/convert.py

```diff
@@ -999,6 +1001,10 @@ def onnx_export_from_model(
     >>> onnx_export_from_model(model, output="gpt2_onnx/")
     ```
     """
+    if check_if_transformers_greater("4.44.99"):
+        raise ImportError(
+            f"ONNX conversion disabled for now for transformers version greater than v4.45, found {transformers.__version__}"
+        )
```

Comment:
disabling onnx export as additional fixes are needed (but don't want to block the latest transformers release for other subpackages) @michaelbenayoun @xenova

Reply:
Ok to me, maybe specify that it's temporary.